tests/ci-arg.earth

```diff
@@ -0,0 +1,8 @@
+VERSION 0.6
+FROM alpine:3.15
+
+ci-test:
+    ARG EARTHLY_CI
+    RUN test "$EARTHLY_CI" = "true" || test "$EARTHLY_CI" = "false"
+    RUN (if [ "$EARTHLY_CI" = "true" ]; then echo UHVzaCBpdCByZWFsIGdvb2Q=; else echo U2hvb3Agc2hvb3AgYmEtZG9vcA==; fi) | base64 -d | tee chorus
+    SAVE ARTIFACT chorus
```

Comment:
is this `SAVE ARTIFACT` needed? I don't see `chorus` being used anywhere in the test.